sssd_test_framework/utils/sssd.py

```diff
@@ -783,3 +784,53 @@ def autofs(self) -> None:
         """
         self.sssd.authselect.select("sssd")
         self.sssd.enable_responder("autofs")
+
+    def proxy(
+        self,
+        proxy: Literal["files", "ldap"] = "files",
+        provider: str | list[str] = "id",
+        proxy_file: str = "system-auth",
+        sever_hostname: str | None = None,
+        domain: str | None = None,
+    ):
+        """
+        :param proxy: ``ldap`` or ``files``
+        :type proxy: str
+        :param provider: SSSD providers (id, auth, chpass, ...)
+        :type provider: str | list[str]
+        :param proxy_file: Config for proxy
+        :type proxy_file: str
+        :param sever_hostname: LDAP (LDAP server hostname)
+        :type sever_hostname: str
+        :param domain: Existing domain name, defaults to None (= default domain)
+        :type domain: str | None, optional
+        """
+        if domain is None:
+            domain = self.sssd.default_domain
+        if domain is None:
+            raise ValueError("No domain specified!")
+
+        proxy_pam_target = proxy_file
+        match proxy:
+            case "ldap":
+                if sever_hostname is None:
+                    raise ValueError("No sever_hostname specified!")
+                proxy = "ldap"
```

Comment:
You are already checking in a condition that proxy is `ldap`, so I don't think this is needed

Reply:
changed